code/modules/events/electrical_storm.dm

```diff
@@ -17,20 +16,20 @@
 
 /datum/round_event/electrical_storm/start()
 	var/list/epicentreList = list()
+	var/list/all_APCs = list()
 
-	for(var/i=1, i <= lightsoutAmount, i++)
-		var/list/possibleEpicentres = list()
-		for(var/obj/effect/landmark/newEpicentre in landmarks_list)
-			if(newEpicentre.name == "lightsout" && !(newEpicentre in epicentreList))
-				possibleEpicentres += newEpicentre
-		if(possibleEpicentres.len)
-			epicentreList += pick(possibleEpicentres)
-		else
-			break
+	for(var/obj/machinery/power/apc/apc in world)

```

Comment:
if your'e goign to bookkeep a list of all APC's make it a global list updated on intialize/destroy